ui/pages/keys/view.go

```diff
@@ -17,3 +27,16 @@ func (m ViewModel) View() string {
 		),
 	)
 }
+
+func renderDeleteConfirmationModal(partKey *api.ParticipationKey) string {
+	modalStyle := lipgloss.NewStyle().
```

Comment:
I would try to center this modal, better yet would be to render it as an overlay